updates/next.json

```diff
@@ -1,8 +1,18 @@
 {
   "stream": "next",
   "metadata": {
-    "last-modified": "2020-06-16T11:51:41Z"
+    "last-modified": "2020-06-16T17:18:09Z"
   },
   "releases": [
+    {
+      "version": "32.20200615.1.0",
+      "metadata": {
+        "rollout": {
+          "start_epoch": 1592335800,
+          "start_percentage": 0.0,
+          "duration_minutes": 4320
```

Comment:
Is there a reason for this long? I think 24h should be enough.

Reply:
we used 72 hours duration for stable. So, made it 72 hours here as well.

Reply:
moved back to 48 hours window